app_legacy/Helpers/database/forum.php

```diff
@@ -145,7 +145,8 @@ function getTopicComments(int $topicID, int $offset, int $count, ?int &$maxCount
         $maxCountOut = (int) $data['COUNT(*)'];
     }
 
-    $query = "SELECT ftc.ID, ftc.ForumTopicID, ftc.Payload, ftc.Author, ftc.AuthorID, ftc.DateCreated, ftc.DateModified, ftc.Authorised, ua.RAPoints
+    $query = "SELECT ftc.ID, ftc.ForumTopicID, ftc.Payload, ftc.Author, ftc.AuthorID, ftc.DateCreated, ftc.DateModified, ftc.Authorised, ua.RAPoints, ua.Created AS AuthorJoined, ua.Permissions as AuthorPermissions,
+                (SELECT COUNT(*) FROM ForumTopicComment WHERE AuthorID = ftc.AuthorID) AS AuthorPostCount
```

Comment:
This adds a not insignificant amount of overhead to the query. I'm not sure if it's enough to ditch it, or even if it's worth it in the first place. 

![image](https://user-images.githubusercontent.com/32680403/236503239-7b3afd7e-7353-447c-8e81-fcdd29030f13.png)
![image](https://user-images.githubusercontent.com/32680403/236503275-b7fad7df-ca30-41f9-ae27-c4c088e53fe2.png)

Maybe we can revisit with denormalized data later?

Reply:
You're right, this should wait for denormalized data. I've stripped post count but left the user role. I will try to get in a better habit of testing the query performance impact of these changes before putting them in PRs. Addressed in https://github.com/RetroAchievements/RAWeb/pull/1534/commits/f708977a67a1788471ac2aee7c6ecc72175d27fb.